content/articles/2018-08-processx-3.2.0.Rmarkdown

```diff
@@ -0,0 +1,339 @@
+special characters), in calls to external programs.
+
+```{r}
+run(px, c("outln", "arg -   with spaces", "outln", "'arg with quote'"))
+```
+
+### Interruption
+
+Unlike `system()` and `system2()`, `processx::run()` is always
+interruptible, you can use the usual interruption key, e.g. ESC in RStudio,
+or CTRL+C in a terminal. On interruption, the external process is
+terminated.
+
+### Spinner
+
+`run()` can show a friendly spinner while the external process is running.
+If the process takes longer then a few second, it is a good idea to use it.
+The spinner is automatically hidden if R is non-interactive:
+
+```r
+run(px, c("sleep", "5"), spinner = TRUE)
+```
+
+### Time limit
+
+You can specify a time limit in `run()`, in seconds, or as a `difftime`
+object:
+
+```{r error = TRUE}
+run(px, c("sleep", "5"), timeout = 1)
+```
+
+`run()` throws an error of class `system_command_timeout_error`, so you
+can easily catch timeouts using `tryCatch()`, if you wish so. By default
+`run()` also throws an error if the system command fails, as indicated by
+its exit status.
+
+```{r, error = TRUE}
+run(px, c("return", "10"))
+```
+
+The `error_on_status` argument can be set to `FALSE` to avoid errors
+for non-zero exit statuses. This can be useful if you anticipate a failure
+and want to handle it without throwing an R error, or if a non-zero exit
+status does not indicate an error for the given program.
+
+### Standard output and error
+
+By default, `run()` collects all standard output and error of the process
+and retuns them in two strings. If desired, it can also echo them to
+the screen while the external process is running. (They are still collected
+and returned, so you can still compute on them.)
+
+```{r}
+outp <- run("ls", "..", echo = TRUE)
+```
+
+### Setting environment variables
+
+You can set environment variables for the external process, via the `env`
+argument. Usually you want to add these variables to those already set in
+the current process, otherwise the external process might fail if some
+essential environment variables (like `PATH`) are not set:
+
+```{r}
+run(px, c("getenv", "FOO"), env = c(Sys.getenv(), FOO = "bar"))
+```
+
+## Advanced usage: background processes
+
+processx really shines when it comes to controlling background processes.
+To start a backgound process, you create an R6 object of class `process`.
+The arguments of `process$new()` mostly correspond to the arguments of
+`run()`.
+
+```{r}
+proc <- process$new(px, c("sleep", "10"))
+proc
+```
+
+`process` objects have methods to query process information and to
+manipulate the subprocess. See `?process` for a complete list of methods.
+
+```{r}
+proc$get_name()
+proc$get_cmdline()
+proc$get_exe()
+proc$is_alive()
+proc$suspend()
+proc$get_status()
+proc$resume()
+proc$get_status()
+proc$kill()
+proc$is_alive()
+proc$get_exit_status()
+```
+
+### Output and polling
+
+The standard output and standard error of a background process is ignored
```

Comment:
I think "are ignored" (since you're referring to both standard output and standard error).